java/src/jmri/jmrix/nce/NceTurnoutManager.java

```diff
@@ -6,11 +6,11 @@
 
 /**
  * Implement turnout manager for NCE systems.
- * <P>
- * System names are "NTnnn", where nnn is the turnout number without padding.
+ * <p>
+ * System names are "NiTnnn", where nnn is the turnout number without padding.
```

Comment:
"NiTnnn" is not a valid format for a system name.  They have to start with a single letter and optional digits.  Is there a reason this had to change?

Reply:
i stands for the digits. Just N is no longer a full description of the system prefix.

Reply:
I've taken out the ``i`` (as well as ``x``) from all system name format Javadoc, added explanation on the user-customizable system prefix where missing. So this seems good to go.